src/amo/components/HomeHeroBanner/index.js

```diff
@@ -52,17 +52,22 @@ export class HomeHeroBannerBase extends React.Component<Props> {
           )}
         </p>
       </HeroSection>,
-      <HeroSection key="hero-6" linkTo="/addon/ublock-origin/">
-        <h3>{i18n.gettext('uBlock Origin')}</h3>
+      <HeroSection key="hero-6" linkTo="/addon/new-tab-override/">
+        <h3>{i18n.gettext('New Tab Override')}</h3>
 
-        <p>{i18n.gettext('Efficient, powerful ad blocker')}</p>
+        <p>
+          {i18n.gettext(
+            'Choose the page you see every time you open a new tab.',
```

Comment:
I know it's in the doc, but we discussed this a few months ago and none of these descriptions should have a period (`.`) at the end of them. You'll see the rest of the file conforms to that.